docs/src/pages/guides/understand-mui-packages/understand-mui-packages.md

```diff
@@ -34,35 +34,48 @@ For this reason, abstracting into smaller packages not only allows MUI to grow o
 
 The packages can be categorized into 3 layers, as shown in the picture below:
 
-<img src="/static/images/packages/mui-packages.png" style="width: 700px; max-width: 100%;" />
+<img src="/static/images/packages/mui-packages.png" style="display: block; width: 448px; max-width: 100%; margin: auto;" />
 
 Let's take a look at each layer to understand how they work together, starting from the bottom:
 
+> 💡 Glossary
+>
+> **install** refers to the action when you run `yarn add $module` or `npm install $module`.
+>
+> **import** means you want to use some APIs from the module by writing `import ... from '$module'`.
+
 ## Styled engines
 
-This layer is specifically related to stylesheets.
-Currently we use `emotion` as the default style library for creating stylesheets.
-In addition, we also provide an adapter for developers who prefer to use styled-components.
+This layer is specifically related to stylesheet generation (CSS-in-js). We have introduced new styled-engines in v5 to unlock more possibilities and enable customization enhancement. Usually, developers does not need to interact with this layer on a daily basis because it is used internally in `@mui/system`.
+
+They come in two packages:
 
-These are the packages in this layer:
+- `@mui/styled-engine` : an [emotion](https://emotion.sh/docs/styled) wrapper
+- `@mui/styled-engine-sc` : a [styled-components](https://styled-components.com/docs/basics#getting-started) wrapper
 
-- `@mui/styled-engine` : an emotion wrapper
-- `@mui/styled-engine-sc` : a styled-components wrapper
-- `@mui/styles` : JSS wrapper (`deprecated`)
+The reason that we create these adapters is to unify the APIs to support both `emotion` and `styled-components`, so that developers can choose what suit them the most.
```

Comment:
```suggestion
These adapters unify the APIs of both `emotion` and `styled-components`, so that developers can choose to use whichever suits them best.
```